nodejs/helpers.js

```diff
@@ -0,0 +1,174 @@
+const fs = require("fs")
+const mongodb = require("mongodb")
+const { ClientEncryption } = require("mongodb-client-encryption")(mongodb)
+const { MongoClient, Binary } = mongodb
+
+module.exports = {
+  readMasterKey: function(path = "./master-key.txt") {
+    return fs.readFileSync(path)
+  },
+  ClientBuilder: class {
```

Comment:
I was finally able to put a finger on this implementation -- it reminds me of a stateful J2EE session bean which I'm not certain is the most appropriate pattern for the task. I wonder if instead it should just be a configuration storage object and return new instances of clients rather than keep references to them. I don't want to generate any additional work for you; just thought I'd mention this.